src/coreclr/tools/Common/TypeSystem/IL/ILImporter.cs

```diff
@@ -324,8 +326,11 @@ private void ImportBasicBlock(BasicBlock basicBlock)
                 StartImportingInstruction();
 
                 ILOpcode opCode = (ILOpcode)ReadILByte();
+                if (opCode == ILOpcode.prefix1)
+                    opCode = (ILOpcode)(0x100 + ReadILByte());
```

Comment:
Nit: There is more instance of the `goto again` pattern in this file. Could you please update it too so it looks the same?

Reply:
Sure!